sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/CosmosClientBuilder.java

```diff
@@ -116,6 +118,15 @@ public CosmosClientBuilder() {
         this.throttlingRetryOptions = new ThrottlingRetryOptions();
     }
 
+    CosmosClientBuilder metadataCaches(CosmosClientMetadataCachesSnapshot metadataCachesSnapshot) {
+        this.state = metadataCachesSnapshot;
+        return this;
+    }
```

Comment:
This should be public instead (see comment above)

Reply:
please see this commit:
https://github.com/Azure/azure-sdk-for-java/pull/18184/commits/2f31869e9c5d84b230f1664070455ef4a20f81f2

The api is internal now and also removed bridge-internal. This is a pattern we can follow elsewhere too.